fuel-vm/src/checked_transaction.rs

```diff
@@ -1125,12 +1125,12 @@ mod tests {
         let fee = TransactionFee::checked_from_tx(&gas_costs, &fee_params, &tx).unwrap();
 
         let min_fee = fee.min_fee();
-        let expected_min_fee = (tx.metered_bytes_size() as u64 * fee_params.gas_per_byte
-            + gas_costs.vm_initialization
+        let expected_min_fee = (tx.size() as u64 * fee_params.gas_per_byte
```

Comment:
I'm switching to `Serialize::size` where available for consistency, but other places must use `metered_bytes_size` since these are different traits to access the same information